packages/not-next/src/hooks/useQuery.ts

```diff
@@ -4,7 +4,7 @@ import { hashMessage } from "@wevm/viem";
 // global cache
 const queryCache = new Map();
 
-export function useQuery<T>(
+export default function useQuery<T>(
```

Comment:
I was not going to use default for any of the hooks. I was thinking in general that stand alone functions would all use normal exports and class definition should use default exports.